src/Extension/AssetTrait.php

```diff
@@ -180,16 +180,16 @@ private function normalizeAsset(FileAssetInterface $asset)
             return;
         }
 
-        $file = $this->getWebDirectory()->getFile($asset->getPath());
+        $app = $this->getContainer();
+
+        $file = $app['filesystem']->getFile("extensions_assets://{$this->getId()}/$path");
```

Comment:
Well, bundles are a user driven thing … the rest are Composer installs and IMHO should follow that naming pattern in this case.